.github/workflows/lint.yaml

```
@@ -122,38 +122,38 @@ jobs:
-    
+
     - name: Setup Node.js
       uses: actions/setup-node@v4
       with:
         node-version: '20.x'
         cache: 'npm'
         cache-dependency-path: frontend/package-lock.json
-    
+
     - name: Setup Python
       uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)